benchmark/initializer.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+gemfile do
+  source "https://rubygems.org"
+  gem "dry-initializer", "~> 3.1"
+  gem "dry-types", "~> 1.5"
+  gem "awesome_print"
+  gem "benchmark-ips"
+  gem "pry"
+  gem "ruby-prof"
+  gem "sorbet"
+  gem "sorbet-runtime"
+  gem "definition", path: File.expand_path("../.", __dir__)
+end
+
+class SorbetUseCase
+  extend T::Sig
+
+  sig { params(phone: String, admin: T::Boolean).void }
```

Comment:
Maybe we should just go with that for the usecases in our services? Its quite simple and you can use it to do some typechecking anywhere in your code, not only for our usecases